include/open62541/server.h

```diff
@@ -607,6 +626,105 @@ UA_EXPORT UA_StatusCode UA_THREADSAFE
 UA_Server_deleteSessionAttribute(UA_Server *server, const UA_NodeId *sessionId,
                                  const UA_QualifiedName key);
 
+/**
+ * .. _roles-access:
+ *
+ * Role Permissions and Access Control
+ * -----------------------------------
+ *
+ * OPC UA uses role-based access control (RBAC). Every session is assigned a set
+ * of roles it represents. Every node in the information contains
+ * RolePermissions that define the rights of each role.
```

Comment:
This does not seem to be correct. When looking into the [specification](https://reference.opcfoundation.org/Core/Part3/v105/docs/5.2.9) it states:

> If not specified, the value of [DefaultRolePermissions](https://reference.opcfoundation.org/search/356?t=DefaultRolePermissions) [Property](https://reference.opcfoundation.org/search/356?t=Property) from the [NamespaceMetadata](https://reference.opcfoundation.org/search/356?t=NamespaceMetadata) [Object](https://reference.opcfoundation.org/search/356?t=Object) associated with the [Node](https://reference.opcfoundation.org/search/356?t=Node) shall be used instead.

My understanding is,  that not every node in the information model contains RolePermissions,

Reply:
Every node can have it (optional).
We would chose to have the overhead for every node.